src/components/UI/Form/DateTimePicker/DateTimePicker.test.tsx

```
@@ -53,17 +51,19 @@ describe('<DateTimePicker />', () => {
     props.field.value = dayjs();
-    render(<DateTimePicker {...props} />);
-    const input = screen.getByRole('textbox');
-    expect(input).toHaveValue();
+    const container = render(<DateTimePicker {...props} />);
+    const dateFrom = container.queryByTestId('Date range');
+    expect(dateFrom).toHaveValue();
   });
 
   it('onChange event will not be called if the prop is not passed', async () => {
     const props = getProps();
     delete props.onChange;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)